### PR TITLE
chore(deps): use postcss-sass instead of postcss-node-sass

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

File: packages/css-dev-tools/postcssPluginConfig.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const plugins = [
   stylelint({ config: styleLintConfig }),
   reporter({ clearReportedMessages: true }),
   cssprepend(`$mozaic-env: ${mozaicEnvScssVar};`),
-  nodeSass({
+  sass({
```
Comment (tiloyi) [inline packages/css-dev-tools/postcssPluginConfig.js:53]: You should remove the option `precision: 7,`, it's not useful when using Dart Sass.

=> More details here: https://github.com/sass/dart-sass#javascript-api

File: packages/css-dev-tools/postcssPluginConfig.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ const plugins = [
 
 const productionPlugins = [
   cssprepend(`$mozaic-env: ${mozaicEnvScssVar};`),
-  nodeSass({
+  sass({
```
Comment (tiloyi) [inline packages/css-dev-tools/postcssPluginConfig.js:71]: You should remove the option `precision: 7,`, it's not useful when using Dart Sass.

=> More details here: https://github.com/sass/dart-sass#javascript-api

File: packages/css-dev-tools/postcssPluginConfig.js (hunk 1 above)
Comment (mohamedMok) [reply packages/css-dev-tools/postcssPluginConfig.js:53]: cb07e64

File: packages/css-dev-tools/postcssPluginConfig.js (hunk 2 above)
Comment (mohamedMok) [reply packages/css-dev-tools/postcssPluginConfig.js:70]: cb07e64

**Review by tiloyi — APPROVED** [review]

**Pull request merged**